Sources/MQ/Errors/Unimplemented.swift

```diff
@@ -118,8 +201,8 @@ public func unimplemented<A1, R>(
 
 /// Convenient placeholder for unimplemented function.
 ///
-/// Placeholder for a function indicating unimplemented part of code.
-/// Terminates process with ``Unimplemented`` error as the cause when the function becomes executed.
+/// Placeholder for a function indicating unimplemented throwing part of code.
+/// Throw ``Unimplemented`` error as the cause.
```

Comment:
Please replace "Throw ``Unimplemented`` error as the cause." with "Throws ``Unimplemented`` error when executed." (applies to all throwing overloads)